manifests/postgres.yaml

```diff
@@ -46,12 +46,12 @@ spec:
           initialDelaySeconds: 60
           timeoutSeconds: 5
           failureThreshold: 6
-        readinessProbe:
+        readiness-probe:
```

Comment:
this is kubernetes manifests, right?

I think the right is readinessProbe, why did you changed it?